token-lending/sdk/src/oracles.rs

```diff
@@ -50,15 +51,33 @@ pub fn get_pyth_price(
         return Err(LendingError::InvalidOracleConfig.into());
     }
 
-    let market_price = if pyth_price.expo >= 0 {
+    let market_price = pyth_price_to_decimal(&pyth_price);
+    let ema_price = {
+        let price_feed = price_account.to_price_feed(pyth_price_info.key);
+        // this can be unchecked bc the ema price is only used to _limit_ borrows and withdraws.
```

Comment:
This seems true but also might be bad UX if you have a really outdated ema price that's never updated and users can never initiate a borrow/withdraw. 

That being said, from my light skimming of pyth code it seems like EMA should always be updated the same slot as the latest valid price update, so not sure if it practically matters

Reply:
> That being said, from my light skimming of pyth code it seems like EMA should always be updated the same slot as the latest valid price update, so not sure if it practically matters

err yeah this was also partially my reasoning, ill update the comment